testing/upload_validation.ts

```diff
@@ -76,8 +84,32 @@ export async function validatePackVersionMetadata(metadata: Record<string, any>)
       validated.error.errors.flatMap(zodErrorDetailToValidationError),
     );
   }
+  const packMetadata = validated.data as PackVersionMetadata;
+  validateColumnFormats(packMetadata.formats);
```

Comment:
I think you probably just want to have zod do this validation, by adding a refiner to the existing `matchers` validator here:

https://github.com/coda/packs-sdk/blob/main/testing/upload_validation.ts#L629

Could be as simple as `matchers: z.array(z.string().regex(<whatever>)),`

Reply:
Or possibly using `refine` to add the extra check that `new RegExp()` works on the value too.

Reply:
Wow, zod does it all. Done!